atlassian_jwt_auth/contrib/django/decorators.py

```diff
@@ -22,7 +22,7 @@ def requires_asap_wrapper(request, *args, **kwargs):
                 verifier=verifier,
                 auth=auth_header,
```

Comment:
Actually I made this commit but I had left my author information set to grant. Sorry!